metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/enrichment/EnricherProvider.java

```diff
@@ -33,16 +35,18 @@ public Enricher create() {
     }
 
     // Create the enrichment client if needed
-    final EnrichmentClient enrichmentClient;
+    RemoteEntityResolver remoteEntityResolver = null;
     if (StringUtils.isNotBlank(enrichmentUrl)) {
-      enrichmentClient = new EnrichmentClient(createRestTemplate(), enrichmentUrl,
-              batchSizeEnrichment);
-    } else {
-      enrichmentClient = null;
+      try {
+        remoteEntityResolver = new RemoteEntityResolver(new URL(enrichmentUrl),
+                batchSizeEnrichment, createRestTemplate());
+      } catch (MalformedURLException e) {
+        e.printStackTrace();
```

Comment:
Again with the error handling... :)
Just convert this to a suitable exception and throw it on.

Reply:
Fixed